docker/Dockerfile.ci_cpu

```diff
@@ -136,3 +139,7 @@ ENV PATH /opt/sccache:$PATH
 # Libxsmm deps
 COPY install/ubuntu_install_libxsmm.sh /install
 RUN bash /install/ubuntu_install_libxsmm.sh
+
+# Hacky workaround (do not push to GitHub) to install the right Numpy version
```

Comment:
Little concered about a comment that says "do not push to Github" in a github PR...

Reply:
Oops! Removed now. For context: this we needed before I realised that `ethos-u-vela` needed to be upgraded, which had a hard requirement on an old numpy. With that, this workaround was not needed anymore.